### DISCUSSION TITLE
Telegram bots

### DISCUSSION THREAD

**Discussion by josephfrances123** [lncrawl/lightnovel-crawler#1193] [Q&A]
I just want to ask if the bots are upgraded as the lightnovel-crawler is updated.. Coz last time I opted to use telegram bots, they won't reply hehe

**Comment by dipu-bd** [comment]
My discord bot `Lightnovel Crawler` is updated as soon as I push a some changes to the master branch. For bots maintained by others, I don't know when and how they update it.
